### PR TITLE
Use localized resources instead of hardcoded strings

### PR DESCRIPTION
**Pull request by 7illusions**

### CODE REVIEW

File: Emby.Mobile.Universal.Core/Emby.Mobile.Universal.Core.nuget.targets [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Condition="'$(NuGetPackageRoot)' == ''">
-    <NuGetPackageRoot>C:\Users\scott\.nuget\packages\</NuGetPackageRoot>
+    <NuGetPackageRoot>C:\Users\Fredrik Zetterlund\.nuget\packages\</NuGetPackageRoot>
```
Comment (ScottIsAFool) [inline Emby.Mobile.Universal.Core/Emby.Mobile.Universal.Core.nuget.targets]: I wonder if we should exclude this file :)

File: Emby.Mobile.Universal/Converters/LocalizationConverter.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+namespace Emby.Mobile.Universal.Converters
+{
+    public class LocalizationConverter : IValueConverter
+    {
+        private static ILocalizedResources _resources;
+        public object Convert(object value, Type targetType, object parameter, string language)
+        {
+            if (_resources == null)
+                _resources = new LocalizedResources();
+            return _resources.GetString(parameter?.ToString());
```
Comment (ScottIsAFool) [inline Emby.Mobile.Universal/Converters/LocalizationConverter.cs]: Why parameter and not value? Are we not using value?

File: Emby.Mobile.Universal/Emby.Mobile.Universal.csproj [hunk 3]
```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Emby.Mobile.Universal</RootNamespace>
     <AssemblyName>Emby.Mobile.Universal</AssemblyName>
-    <DefaultLanguage>en-GB</DefaultLanguage>
+    <DefaultLanguage>en-US</DefaultLanguage>
```
Comment (ScottIsAFool) [inline Emby.Mobile.Universal/Emby.Mobile.Universal.csproj]: Git

File: Emby.Mobile.Universal/Services/AppServices.cs [hunk 4]
```diff
@@ -57,10 +58,12 @@ public static void Create()
             var mbLogger = new MBLogger(logger);
             var device = new Device();
             var network = new NetworkConnection();
+            var localizedResources = new LocalizedResources();
```
Comment (ScottIsAFool) [inline Emby.Mobile.Universal/Services/AppServices.cs]: This doesn't need creating here, you can do it the same as you have done in line 91. Reason for the others being done like that is they are used further down and it was quicker than getting them from the cache.

File: Emby.Mobile.ViewModels/ViewModels/ChooseServerViewModel.cs [hunk 5]
```diff
@@ -62,7 +62,7 @@ public RelayCommand RefreshCommand
                 return;
             }
 
-            SetProgressBar("**Getting servers...");
+            SetProgressBar(Services.LocalizedResources.GetString("Getting servers..."));
```
Comment (ScottIsAFool) [inline Emby.Mobile.ViewModels/ViewModels/ChooseServerViewModel.cs]: This should be the string name, not the actual string (see why I *hate* this approach).

File: Emby.Mobile.ViewModels/ViewModels/MainViewModel.cs [hunk 6]
```diff
@@ -8,6 +8,6 @@ public MainViewModel(IServices services) : base(services)
         {
         }
 
-        public string ConnectedTo => $"Connected to {Services.ServerInfo?.ServerInfo?.Name}";
+        public string ConnectedTo => string.Format(Services.LocalizedResources.GetString("LabelServerConnected"),Services.ServerInfo?.ServerInfo?.Name);
```
Comment (ScottIsAFool) [inline Emby.Mobile.ViewModels/ViewModels/MainViewModel.cs]: Code style, please put a space after the comma :)

**Comment by ScottIsAFool** [comment]
I'm wondering if it's worth putting a property in ViewModelBase for Localisation, rather than having to do Services.Localisation all the time, just messes up the code, especially as we're having to pass in string names to get the resources, can just make it really long winded code.

File: Emby.Mobile.Universal/Converters/LocalizationConverter.cs (hunk 2 above)
Comment (7illusions) [inline Emby.Mobile.Universal/Converters/LocalizationConverter.cs]: Value and x:Bind won't work.

Comment (ScottIsAFool) [inline Emby.Mobile.Universal/Converters/LocalizationConverter.cs]: Fair point.

File: Emby.Mobile.ViewModels/ViewModels/ChooseServerViewModel.cs (hunk 5 above)
Comment (7illusions) [inline Emby.Mobile.ViewModels/ViewModels/ChooseServerViewModel.cs]: And this is why we review it. :)

**Pull request merged**